mass-deployer/cmd/main.go

```diff
@@ -0,0 +1,91 @@
+package cmd
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/workloads"
+	"github.com/threefoldtech/tfgrid-sdk-go/mass-deployer/internal/parser"
+	deployer "github.com/threefoldtech/tfgrid-sdk-go/mass-deployer/pkg/mass-deployer"
+)
+
+func Execute() {
+	var configFile string
+	flag.StringVar(&configFile, "c", "", "path to config file")
+	flag.Parse()
+
+	if configFile == "" {
+		log.Fatal("couldn't locate config file")
+	}
+
+	cfg, err := parser.ParseConfig(configFile)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	d, err := deployer.NewDeployer(cfg)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 100*time.Minute)
+	defer cancel()
+
+	groupsNodes := map[string][]int{}
+
+	pass := map[string]error{}
```

Comment:
I don't understand the pass map

Reply:
I've changed it to a list of groups names for now